src/test/java/com/zufar/icedlatte/favorite/converter/FavoriteItemDtoConverterTest.java

```diff
@@ -0,0 +1,39 @@
+package com.zufar.icedlatte.favorite.converter;
+
+import com.zufar.icedlatte.favorite.dto.FavoriteItemDto;
+import com.zufar.icedlatte.favorite.entity.FavoriteItemEntity;
+import com.zufar.icedlatte.product.converter.ProductInfoDtoConverter;
+import com.zufar.icedlatte.product.entity.ProductInfo;
+import org.junit.jupiter.api.BeforeEach;
+import org.mapstruct.factory.Mappers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+
+import java.util.UUID;
+
+public class FavoriteItemDtoConverterTest {
+
+    private FavoriteItemDtoConverter converter;
+    private ProductInfoDtoConverter productInfoDtoConverter;
+
+    @BeforeEach
+    void setup() {
+        converter = Mappers.getMapper(FavoriteItemDtoConverter.class);
+        productInfoDtoConverter = Mappers.getMapper(ProductInfoDtoConverter.class);
+    }
+
+    @Test
+    @DisplayName("Convert FavoriteItemEntity to FavoriteItemDto")
+    void toDtoTest() {
```

Comment:
At the moment this test doesn't make any sense (because assert should be improved), and I suggest you commit it in another PR 👍

Reply:
As the initial task was to add tests for all mappers I'd better leave it here.

Reply:
Ok, then could you please finish this test? I just want to avoid a situation when we end up having tests that don't check anything 😸

Reply:
In this test still remains the problem with different types of sets that I can't compare now so ok I'll remove this test.